pallets/crowdloan-rewards/src/lib.rs

```diff
@@ -167,7 +131,12 @@ pub mod pallet {
 				.ok()
 				.ok_or(Error::<T>::WrongConversionU128ToBalance)?;
 
-			let reward_per_block = info.total_reward / reward_period;
+			ensure!(
```

Comment:
didn't see one month lock

Reply:
Use the Setting struct to set the locked time: claim must be between start_block and end_block.
https://github.com/polkafoundry/newpolka/pull/55/commits/2ab33c221080d49511dd1ad89c9e64ffa03503e4#diff-2bbae40c9c7fdfeeb0b1708d368844303a595083bc948ccd684b577e5d13f6ebR150